rebar.config

```
@@ -1,11 +1,12 @@
 {require_min_otp_vsn, "21"}.
 
 {deps, [
-    {euneus, "0.6.0"}
+    {euneus, "~>2.3"},
+    {json_polyfill, "0.1.3"}
```

```suggestion
    {json_polyfill, "0.1.4"}
```

Please use this version, and you are good to go with OTP-27.

---

Great!  Giving it a go :-)

---

Looking at your change, a pity that such a hack is needed for a OTP version-dependency.

---

Yes :sweat_smile: 

Having a min and max OTP version option for such a case would be nice.